File: Python-Basics/first-steps/yard_greening.py
square_meters = float(input())

full_price = square_meters * 7.61
discount = 0.18 * full_price
real_price = full_price - discount

print(f'The final price is: {real_price:.2f} lv.')
print(f'The discount is: {discount:.2f} lv.')

File: CPP-Basics/Yard Greening.cpp
#include <iostream>

using namespace std;

int main () {

    double sqMts;
    cin >> sqMts;

    ///price per yard sq mt 7.61
    ///discount 18%

    double fullPrice = sqMts * 7.61;
    double discount = fullPrice * 18/100;

    double finalPrice = fullPrice - discount;

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << "The final price is: " << finalPrice << " lv." << endl;
    cout << "The discount is: " << discount << " lv." << endl;

    return 0;
}
